src/Components/GrowthChart/GrowthChartBuilder/GrowthChartBuilder.tsx

```diff
@@ -14,16 +15,19 @@ export const GrowthChartBuilder = ({
             data: dataSetValues.map((entry) => entry[key]),
             borderWidth: 0.9,
             borderColor: chartLineColorPicker(key),
+            label: key,
         })),
     };
 
-    const options = {
+    const options: any = {
         elements: { point: { radius: 0, hoverRadius: 0 } },
         plugins: { legend: { display: false } },
         scales: {
             x: { title: { display: true, text: i18n.t(`age (${dataSetMetadata.unit})`) } },
             y: { title: { display: true, text: dataSetMetadata.yaxis } },
         },
+        layout: { padding: { right: 75 } },
+        animation: { onProgress: annotateLineEnd },
```

Comment:
After removing the any type for options, you will get an type error in option for the animation parameter. You can solve this by using ES6+ syntax, and defining type for `animation`
```suggestion
    animation: {
    onProgress(animation: AnimationForChart) {
        annotateLineEnd(animation);
    }
}
```

Reply:
Maybe add this to `chartDataTypes.ts``
```suggestion
import { Animation, Chart } from 'chart.js';

interface AnimationWithChart extends Animation {
    chart?: Chart;
}
```